vagrant.rb

```diff
@@ -56,10 +55,37 @@
 $windows2008 = "opentable/win-2008-enterprise-amd64-nocm"
 $windows2012r2 = "opentable/win-2012r2-standard-amd64-nocm"
 
+# Format pf_name => { 'pf_id' => 0, 'last_host_id' => 0, 'host_list' => [ 'host1', 'host2' ] }
+$platforms = {
+}
+$last_pf_id = 0
+
+def configure_box(config, os, pf_name, host_name, 
+                  setup:'empty', version:nil, server:'', host_list:'',
+                  windows_plugin:false, advanced_reporting:false,
+                  ncf_version:nil, cfengine_version:nil, ram:nil
+                 )
+  pf = $platforms.fetch(pf_name) { |key| 
```

Comment:
As i understand the goal is to replace all calls to configure to configure_box in vagrantfile in another pull request ?

But,  unless you start all platforms, the platforms variable will always start from scratch and so have id 0 for all platform ?